Cmdr/BuiltInTypes/Player.lua

```diff
@@ -4,6 +4,13 @@ local Players = game:GetService("Players")
 local function ShorthandSingle (text, executor)
 	if text == "." or text == "me" then
 		return {executor}
+	elseif text == "?" then
+		local players = Players:GetPlayers()
+		if #players <= 1 then
+			return players
+		else
+			return {players[math.random(1, #players)]}
+		end
```

Comment:
```suggestion
		return {players[math.random(1, #players)]}
```

I think the performance gain is not worth the added complexity here